Source/JavaScriptCore/bytecode/Repatch.cpp

```diff
@@ -885,7 +885,7 @@ static InlineCacheAction tryCachePutBy(JSGlobalObject* globalObject, CodeBlock*
                     }
                 }
 
-                newCase = ProxyableAccessCase::create(vm, codeBlock, AccessCase::Replace, propertyName, slot.cachedOffset(), oldStructure, ObjectPropertyConditionSet(), isProxy);
+                newCase = AccessCase::createReplace(vm, codeBlock, propertyName, slot.cachedOffset(), oldStructure, isProxy);
```

Comment:
Can you put a comment in the commit message about why this is a bug and the fix here?

Reply:
This was a bug?

Reply:
It was a bug, but no effect on correctness since we are using `virtual` to get a watchpoint-set from ProxyableAccessCase. It is just waste of memory, but that's it.